app/models/keyword.rb

```diff
@@ -7,11 +7,20 @@ class Keyword < ApplicationRecord
 
   enum status: { in_progress: 0, completed: 1, failed: 2 }
 
+  has_many :links, dependent: :destroy
+
   def update_status(status)
     update(status: status)
   end
 
-  def add_html(html)
-    update(html: html, status: :completed)
+  def update_data(html:, parse_result:)
```

Comment:
```suggestion
  def update_result(html:, parse_result:)
```

Reply:
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)